src/main/java/com/github/pires/hazelcast/HazelcastDiscoveryController.java

```diff
@@ -43,22 +55,24 @@
 
   @JsonIgnoreProperties(ignoreUnknown = true)
   static class Address {
-
     public String IP;
   }
 
   @JsonIgnoreProperties(ignoreUnknown = true)
   static class Subset {
-
     public List<Address> addresses;
   }
 
   @JsonIgnoreProperties(ignoreUnknown = true)
   static class Endpoints {
-
     public List<Subset> subsets;
   }
 
+  private static String getServiceAccountToken() throws IOException {
+    String file = "/var/run/secrets/kubernetes.io/serviceaccount/token";
```

Comment:
It's automatically included in all pods now.  See:

https://github.com/GoogleCloudPlatform/kubernetes/pull/7101